backend/apis/parse-docket-pdf.utils.js

```diff
@@ -72,15 +80,25 @@ function parseCharges(lines, sections) {
         .slice(0, lastNumericIndex + 1)
         .trim();
     }
-  });
+  }
+
+  walkSection(lines, sections, "CHARGES", addDispositionToCharges);
 
   return charges;
 }
 
+function parseAccountSummary(lines, sections) {
+  const accounts = [];
+
+  walkSection(lines, sections, "ACCOUNT SUMMARY", line => {
+    accounts.push(line);
```

Comment:
Ideally we'd do better than just pushing the raw text into an array, but only extract the relevant values from the account summary section. Is that something you're planning on working on in a future PR?

Reply:
Yeppers. This is why I was asking about a general parsing strategy. It turns out that if you look at the weeds of the PROCEEDINGS, any payment or fine balance = 0 should appear there as well. We could get a quick answer to "Does this person have a 0 account balance on all accounts?" by parsing through the ACCOUNT SUMMARY section but my fear is that we'll need to parse the entire PROCEEDINGS section anyway, so why not answer that question there? Or are we planning on having multiple functions passing over a fully parsed document perhaps doing double duty? Or do we just clean up the data here, saving everything, and test it's qualification status somewhere else? Or what? These questions need to be answered for the team before any further time is wasted on a more mature parse of the account. Hence, I just sent the whole bloody thing back to the client for now.

Reply:
Let's figure out the answers to those questions in future PRs. I'm okay with merging in what you have here for the account summary and then iterating on it in future PRs.